### DISCUSSION TITLE
Mutations and Pages

### DISCUSSION THREAD

**Discussion by ma00** [torchbox/wagtail-grapple#207] [Q&A] [answered]
Hi, is it possible to use graphql to create a new wagtail page and save it as a draft?

**Comment by zerolab** [comment, marked as answer]
Not out of the box, no.
But you could use https://wagtail-grapple.readthedocs.io/en/latest/general-usage/hooks.html#mutation

**Reply by ma00** [reply]
thanks!

**Comment by ma00** [comment]
I got the mutation to work, however I have some doubts about Streamfield compatible data.
Any suggestions about arguments for the body?

**Reply by zerolab** [reply]
I suggest you join the [Wagtail Slack](https://github.com/wagtail/wagtail/wiki/Slack) and get help from the wider community on that particular bit.
Some links to get you started: https://stackoverflow.com/a/46798240/201316 and https://www.caktusgroup.com/blog/2019/09/12/wagtail-data-migrations/
